newIDE/app/src/ObjectEditor/Editors/TextEditor.js

```diff
@@ -154,7 +162,61 @@ export default class TextEditor extends React.Component<EditorProps, void> {
                 </Button>
               </Tooltip>
             </ButtonGroup>
-          </Line>
+            <ButtonGroup size="small">
+              <Tooltip title={<Trans>Align text on the top</Trans>}>
```

Comment:
Let's remove the tooltip for these buttons. They should be self explanatory and the object panel will make them even less useful.